File: app/src/main/java/jetpack/zmkj/com/jetpack/ui/main/MainActivity.java
package jetpack.zmkj.com.jetpack.ui.main;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import jetpack.zmkj.com.jetpack.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_activity);
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction().replace(R.id.container, MainFragment.newInstance()).commitNow();
        }
    }
}

File: app/src/main/java/jetpack/zmkj/com/jetpack/ui/main/MainActivity.kt
package jetpack.zmkj.com.jetpack.ui.main

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import jetpack.zmkj.com.jetpack.R

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_activity)
        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                    .replace(R.id.container, MainFragment.newInstance())
                    .commitNow()
        }

    }

}